src/UI/AppLayout.jsx

```diff
@@ -16,11 +11,13 @@ function AppLayout() {
   };
 
   return (
-    <StyledAppLayout>
-      <Header toggleModal={toggleModal} />
+    <div className="box-border w-full">
+      <Header toggleModal={toggleModal} className="sticky top-0" />
```

Comment:
Try to avoid adding className to React components. In this case, the proper way would be to put these two classes to the root element in the `Header` component.

Ideally, we pass only the props that a component needs (which `className` can also be, but it's a rare case)